doc/developer-install.md

```diff
@@ -29,11 +41,12 @@ none /cygdrive cygdrive binary,posix=0,user,noacl 0 0
 
 Close and re-open Cygwin Terminal.
 
+#### Branding
```

Comment:
The Branding heading should go a couple lines further down, after the lines about an issue.

Just above "In order to rebrand a build..."